api/src/main/webapp/static/aou-file-tree-policy-extension.js

```diff
@@ -42,7 +42,7 @@ define([
 
     const viewableFileExtenstions = ['ipynb', 'txt', 'sh'];
     // Clicking the view button will
-    // 1) Open All Jupyter files in a new tab.
+    // 1) Open All Jupyter, text or shell files in a new tab.
```

Comment:
`isJupyterFile` is no longer a correct name, then

Reply:
Changed to openInANewTab